src/renderer/components/App.tsx

```diff
@@ -64,6 +66,19 @@ export default class App extends React.Component<any, undefined> {
         ));
     }
 
+    public cleanTabFile(files: File[]): File[] {
+        const tab: File[] = [];
+
+        files.map((file) => {
+            if (file.name.search(".epub") !== -1 ||
+                file.name.search(".opds") !== -1 ||
```

Comment:
1) Note that OPDS feeds do not necessarily end with the `.opds` file extension. For example: https://www.feedbooks.com/catalog.atom
2) I do not think OPDS feeds can be drag-dropped onto the Thorium UI anyway, see error:
```
[2]   readium-desktop:main#redux/sagas/api TypeError: Cannot read property 'resources' of null
[2]   readium-desktop:main#redux/sagas/api     at PublicationViewConverter.convertDocumentToView (/Volumes/500GB/Code/readium-desktop/dist/main.js:2891:47)
[2]   readium-desktop:main#redux/sagas/api     at newDocs.map (/Volumes/500GB/Code/readium-desktop/dist/main.js:2494:67)
[2]   readium-desktop:main#redux/sagas/api     at Array.map (<anonymous>)
[2]   readium-desktop:main#redux/sagas/api     at PublicationApi.<anonymous> (/Volumes/500GB/Code/readium-desktop/dist/main.js:2493:28)
[2]   readium-desktop:main#redux/sagas/api     at Generator.next (<anonymous>)
[2]   readium-desktop:main#redux/sagas/api     at fulfilled (/Volumes/500GB/Code/readium-desktop/dist/main.js:2400:58)
[2]   readium-desktop:main#redux/sagas/api     at process._tickCallback (internal/process/next_tick.js:68:7) +0ms
```